### PR TITLE
muninlite: update to 2.0.1

### PR DESCRIPTION
**Pull request by sumpfralle**
Maintainer: @jmccrohan 
Compile tested: ar71xx, 19.07.2
Run tested: ar71xx, 19.07.2

Description:
* follow upstream ressources to github
* rename /usr/sbin/munin-node to /usr/sbin/muninlite
  (following the change of upstream)
* change plugin directory from /usr/sbin/munin-node-plugin.d/
  to /etc/munin/plugins (compatible to upstream / munin-node)
* all patches (except one OpenWrt-specific patch) were merged
  upstream

Signed-off-by: Lars Kruse <devel@sumpfralle.de>

### CODE REVIEW

**Comment by sumpfralle** [comment]
@jmccrohan: any comments?

File: admin/muninlite/Makefile [hunk 1]
```diff
@@ -24,7 +24,7 @@ define Package/muninlite
   CATEGORY:=Administration
   DEPENDS:=+xinetd
   TITLE:=Munin node implemented in shell
-  URL:=http://sourceforge.net/projects/muninlite/
+  URL:=https://github.com/munin-monitoring/muninlite
   PKG_MAINTAINER:=Jonathan McCrohan <jmccrohan@gmail.com>
```
Comment (neheb) [inline admin/muninlite/Makefile:28]: Wrong variable and wrong place.

Should be before PKG_LICENSE.

Comment (sumpfralle) [reply admin/muninlite/Makefile]: Thanks, done.

**Comment by neheb** [comment]
Is the Build/Compile section needed?

**Comment by sumpfralle** [comment]
> Is the Build/Compile section needed?

Indeed. I removed `Build/Configure` and `Build/Compile` now.

From my point of view it should be ready now.

**Comment by diizzyy** [comment]
Is there a reason why the hostname patch is still in?
```cat /proc/sys/kernel/hostname``` returns hostname on my systems at least

**Comment by sumpfralle** [comment]
> Is there a reason why the hostname patch is still in?

Just my lack of attention ...

Thanks for the hint. I removed it.

**Comment by soif** [comment]
I've just fixed munin NOT starting in #13505

**Pull request merged**